src/main/steps/c100-rebuild/c100sequence.ts

```diff
@@ -400,6 +401,11 @@ export const C100Sequence: Step[] = [
     showInSection: Sections.C100,
     getNextStep: () => C100_MIAM_MEDIATOR_CONFIRMAION,
   },
+  {
```

Comment:
@monalisasarkar1 Always add these lines at the end so that we will have less conflicts.